scylla-server-rust/src/controllers/mod.rs

```diff
@@ -4,4 +4,5 @@ pub mod driver_controller;
 pub mod location_controller;
 pub mod node_controller;
 pub mod run_controller;
+pub mod send_config_controller;
```

Comment:
So the naming for this is kind of weird cause this controller works differently from the rest of them since it sends commands to the car instead of receiving them. I'd name this car_command_controller or config_command_controller. We want the names to be object oriented instead of action oriented if that makes sense